peaseq-case.wdl

```diff
@@ -459,52 +582,53 @@ workflow peaseq {
-                    default_location='SAMPLE/' + sub(basename(fastqpair.left),'_R?[12]_....f.*q.gz|_R?[12].f.*q.gz','') + '/QC/SummaryStats'
+            call util.basicfastqstats as indv_R1_bfs { input:
+                fastqfile = fastqpair.left,
+                default_location = "SAMPLE/" + sub(basename(fastqpair.left), "_R?[12]_....f.*q.gz|_R?[12].f.*q.gz",
+                    "") + "/QC/SummaryStats",
             }
-            call mapping.mapping as indv_PE_mapping {
-                input :
-                    fastqfile=fastqpair.left,
-                    fastqfile_R2=fastqpair.right,
-                    metricsfile=indv_R1_bfs.metrics_out,
-                    insert_size=insertsize,
-                    strandedness=strandedness,
-                    index_files=actual_bowtie_index,
-                    blacklist=blacklist,
-                    paired_end=true,
-                    default_location='SAMPLE/' + sub(basename(fastqpair.left),'_R?[12]_....f.*q.gz|_R?[12].f.*q.gz','') + '/BAM_files'
-            }
-
-            call fastqc.fastqc as indv_PE_bamfqc {
-                input :
-                    inputfile=indv_PE_mapping.sorted_bam,
-                    default_location='SAMPLE/' + sub(basename(fastqpair.left),'_R?[12]_....f.*q.gz|_R?[12].f.*q.gz','') + '/QC/FastQC'
+            call mapping.mapping as indv_PE_mapping { input:
+                fastqfile = fastqpair.left,
+                fastqfile_R2 = fastqpair.right,
+                metricsfile = indv_R1_bfs.metrics_out,
+                insert_size = insertsize,
+                strandedness = strandedness,
+                index_files = actual_bowtie_index,
+                blacklist = blacklist,
+                paired_end = true,
+                default_location = "SAMPLE/" + sub(basename(fastqpair.left), "_R?[12]_....f.*q.gz|_R?[12].f.*q.gz",
+                    "") + "/BAM_files",
             }
 
-            call runspp.runspp as indv_PE_runspp {
-                input:
-                    bamfile=select_first([indv_PE_mapping.bklist_bam, indv_PE_mapping.sorted_bam])
+            call fastqc.fastqc as indv_PE_bamfqc { input:
+                inputfile = indv_PE_mapping.sorted_bam,
+                default_location = "SAMPLE/" + sub(basename(fastqpair.left), "_R?[12]_....f.*q.gz|_R?[12].f.*q.gz",
+                    "") + "/QC/FastQC",
             }
 
-            call peaseq_util.pe_bamtobed as indv_PE_bamtobed {
-                input:
-                    bamfile=select_first([indv_PE_mapping.bklist_bam, indv_PE_mapping.sorted_bam])
+            call runspp.runspp as indv_PE_runspp { input: bamfile = select_first([
+                indv_PE_mapping.bklist_bam,
+                indv_PE_mapping.sorted_bam,
+            ]) }
+
+            call peaseq_util.pe_bamtobed as indv_PE_bamtobed { input: bamfile = select_first(
+            [
+                indv_PE_mapping.bklist_bam,
+                indv_PE_mapping.sorted_bam,
+            ]) }
```

Comment:
this is funky. Is this raw `format` output or was this changed by hand?

Reply:
That's the raw format output.